pkg/socketlb/socketlb.go

```diff
@@ -91,7 +91,7 @@ func Enable(sysctl sysctl.Sysctl) (err error) {
 	})
 	var ve *ebpf.VerifierError
 	if errors.As(err, &ve) {
-		if _, err := fmt.Fprintf(os.Stderr, "Verifier error: %s\nVerifier log: %v\n", err, ve); err != nil {
+		if _, err := fmt.Fprintf(os.Stderr, "Verifier error: %s\nVerifier log: %+v\n", err, ve); err != nil {
```

Comment:
This may dump megabytes of logs into the output. `%-100v` will give you the last 100 lines, would that be sufficient as well?